src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -43,8 +48,24 @@ case class WorkflowDescriptor(id: WorkflowId,
   val name = namespace.workflow.unqualifiedName
   val actualInputs: WorkflowCoercedInputs = coercedInputs ++ declarations
   val props = sys.props
+  val relativeWorkflowRootPath = s"$name/$id"
+  val wfOutputsRoot = workflowOptions.get("outputs_path").toOption
   lazy val fileHasher: FileHasher = { wdlFile: WdlFile => SymbolHash(ioManager.hash(wdlFile.value)) }
 
+  // GCS FS with the workflow working directory as root
+  val gcsFilesystem = for {
+    interface <- gcsInterface.toOption
```

Comment:
Yes I could actually keep it a Try, I made it an option because the `toPath` method takes an `Option[GcsFileSystem]` but it could as well take a `Try[GcsFileSystem]`